vpp-manager/vpp_runner.go

```diff
@@ -807,6 +842,38 @@ func (v *VppRunner) runVpp() (err error) {
 		<-vppDeadChan
 		return errors.Wrap(err, "Error configuring VPP")
 	}
+	v.GetPhysicalNets()
+	for phyNet := range v.physicalNets {
```

Comment:
Instead of computing the whole list and then allocating the physicalNetworks, we could have this logic happen opportunistically, i.e. each time we see a physicalNetworkName and we don't know about it. We create it on the fly.
This can be useful when we move to a more dynamic setup (i.e. uplinks come & go).